migtests/tests/pg/basic-non-public-live-test/validateAfterChanges

```diff
@@ -73,6 +73,10 @@ def change_expected_values():
 def migration_completed_checks_yb():
 	print("Running tests on YB")
 	yb.run_checks(migration_completed_checks)
+	yb.run_checks(YB_specific_checks)
```

Comment:
We should add it for the dependent-ddls and codependent schemas tests. Those will cover nice scenarios. We can add more later.

Reply:
i didn't get this comment @shubham-yb.
I have removed the checks from `validateAfterChanges` for now

Reply:
NVM, I can add it in a separate PR.